test/test-cases/regression/action-ctl_rule_remove_by_id.json

```diff
@@ -28,15 +28,15 @@
     },
     "rules":[
         "SecRule REQUEST_FILENAME \"@endsWith /wp-login.php\" \"id:9002100,phase:2,t:none,nolog,pass,ctl:ruleRemoveById=1\"",
-        "SecRule ARGS \"@contais whe\" \"id:1,phase:3,t:none,nolog,pass,tag:'CRS'\""
+        "SecRule ARGS_NAMES \"@contains whee\" \"id:1,phase:3,t:none,nolog,pass,tag:'CRS'\""
```

Comment:
My first thought "Why did it worked before" ... and then I see this "expected" block wich rely on debug output analysis. Maybe not for this PR, but I think we should at least for all tests do not use the debug output at all but rely on observable behaviour. Like making this rule a

```
SecRule REMOTE_ADDR "@unconditionalMatch" "id:1,phase:2,deny,status=403"
```
and then having 

```
    "expected":{
      "http_code": 200
    },
```

Reply:
My first idea was to make a small cleanup with minimal modifications.

Yes, this test works because it checks only the debug log message - which is problematic because there could be a hidden error (see #3053 - this test would have failed there).

But you're right, if we touch a test, should make it more cleaner.

See my new commit [@80e244](https://github.com/owasp-modsecurity/ModSecurity/pull/3080/commits/80e244388438b729e7cba4d8a6e48ac81be2b1d8) which contains this condition too.

Reply:
@airween This new condition in expected make only sense if the rule we disable would return another thing, right? For example `deny,status=403`.

Same for the other tests here. 
As I mentioned, not really needed as part of this PR, but adding the test for `200` without changing the rules does not really make sense to me.

Reply:
Ah, right, I reverted my last commit.